src/domains/profile/components/SelectRecipient/SelectRecipient.tsx

```diff
@@ -132,7 +134,14 @@ export const SelectRecipient = React.forwardRef<HTMLInputElement, SelectRecipien
 								),
 							},
 							start: {
-								content: <ProfileAvatar address={selectedAddress} />,
+								content: (
+									<div className="flex items-center">
+										<ProfileAvatar address={selectedAddress} />
+										{!!selectedWalletAlias && (
+											<span className="font-semibold ml-2">{selectedWalletAlias}</span>
```

Comment:
Hey @alesmit we might as well truncate it in the end so that it could align the wallet name/address with the sender field :hmhm:

```suggestion
											<TruncateEnd
												className="font-semibold ml-2"
												text={selectedWalletAlias}
												showTooltip
											/>

											/>

```